src/js/components/MarathonTaskDetailsList.js

```diff
@@ -58,6 +58,14 @@ class MarathonTaskDetailsList extends React.Component {
     return task.status;
   }
 
+  getTimeField(time) {
+    return (
+      <time dateTime={time} title={time}>
+        {new Date(time).toLocaleString()}
```

Comment:
Looks good, happy to keep consistent with Marathon for now. This might be something that needs a deeper discussion outside of this PR. `How do we want to handle dates and times?`

We have a few options bouncing around. User time zone, server time zone, UTC. Absolute or relative. I'm not sure how consistent we are. 

One idea is showing the alternate on hover (either tooltip or title attribute). Another is clicking a date/time to switch. cc @air